packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -328,6 +325,124 @@ const EDiscovery = WebexPlugin.extend({
     }
   },
 
+  @waitForValue('@')
+  @oneFlight({keyFactory: (reportId, options) => createOneFlightKey(reportId, options)})
```

Comment:
I'll check it out, if it doesn't work that means other APIs are affected...

Reply:
It was working correctly, I've modified createOneFlightKey because of the new types option for getContentContainers, and added new unit tests to ensure oneFlight behaves correctly.

Reply:
👍 

Actually missed `createOneFlightKey(..)` and read the code as using only the `options` object